eng/pipelines/templates/jobs/live.tests.yml

```diff
@@ -77,6 +77,13 @@ jobs:
       container: $[ variables['Container'] ]
 
     steps:
+      - ${{ if not(contains(variables['Build.DefinitionName'], '-pr - ')) }}:
+        - template: /eng/common/pipelines/templates/steps/sparse-checkout.yml
+          parameters:
+              Paths:
+                - "/*"
```

Comment:
You could probably narrow this down further but it may be diminishing returns.

Reply:
Yeah, I think the largest win is coming from SessionRecords. When you go deeper project-project dependencies start to matter and things get messy.